hawc/apps/study/templates/study/attachment_form.html

```diff
@@ -1,6 +1,5 @@
 {% extends 'assessment-rooted.html' %}
 
-{% load add_class %}
 {% load crispy_forms_tags %}
```

Comment:
I think you suggested making `crispy_forms_tags` a builtin too - what do you think?